diff --git a/node_modules/tap-parser/node_modules/minipass/index.js b/node_modules/tap-parser/node_modules/minipass/index.js
index e8797aa..822f1a4 100644
--- a/node_modules/tap-parser/node_modules/minipass/index.js
+++ b/node_modules/tap-parser/node_modules/minipass/index.js
@@ -625,7 +625,7 @@ module.exports = class Minipass extends Stream {
     this[DESTROYED] = true
 
     // throw away all buffered data, it's never coming out
-    this.buffer.length = 0
+    // this.buffer.length = 0
     this[BUFFERLENGTH] = 0
 
     if (typeof this.close === 'function' && !this[CLOSED])
